Dockerfile

```diff
@@ -2,15 +2,9 @@ FROM python:3.6.8
 
 ENV PYTHONUNBUFFERED 1
 ENV SECRET_KEY="an insecure development secret"
-ENV FRONTEND_DIR="/opt/PyTexas2019/"
 
-COPY requirements.txt /tmp/requirements.txt
-RUN pip install -r /tmp/requirements.txt
-
-COPY Django-Conference /opt/DjangoConference
-RUN pip install -e /opt/DjangoConference
-
-COPY PyTexasBackend /opt/PyTexasBackend
-WORKDIR /opt/PyTexasBackend
+COPY PyTexas /opt/PyTexas
+WORKDIR /opt/PyTexas
 
+RUN pip install -r requirements.txt
```

Comment:
Keeping this before the `COPY PyTexas` command allows the container to rebuild without having to reinstall all the dependencies everytime our codebase changes.

Reply:
got it 👍